### ISSUE TITLE
I just want to say

### ISSUE THREAD

**Issue by nickeubank** [kylebarron/stata_kernel#300]
Kyle, you are my hero. Way to drag Stata into the modern data science ecosystem. 

That is all. :)

**Comment by lordflaron** [comment]
I second that!

**Comment by mcaceresb** [comment]
Kyle appreciation thread! ^^

**Comment by kylebarron** [comment]
Haha on issue #300 as well. I'd feel a bit better about everything if I were doing this for an open source project. I'm a little sad how all the value accrues to a for-profit company. Alas, hopefully this project at least helps me to land a software job!

**Comment by nickeubank** [comment]
I mean, I think you're also making lots of (non-profit) researchers more productive, so there's a pro-social side! :)

**Comment by kylebarron** [comment]
True. Alas, sorry to break it to you but I'm leaving the Economics world and won't be putting in lots of effort to maintain these projects.

**Comment by jeremysze** [comment]
Go for it! Thanks for this great piece of work!

**Comment by emaadmanzoor** [comment]
This tool may just become the economists' gateway drug to Python!
